src/Router.fs

```diff
@@ -228,7 +235,14 @@ type Router =
 
     static member inline format([<ParamArray>] xs: string array) =
         Router.encodeParts (List.ofArray xs) RouteMode.Hash
-
+    
+    static member inline format([<ParamArray>] xs: string array, queryString: (string * string) list) : string =
```

Comment:
`[<ParamArray>]` doesn't really do anything unless the argument is the last one in the function so that it can accept as many values as needed without conflict with other parameters. Can you please remove the usage of `[<ParamArray>]` when it is not applied to the last parameter? 🙏

Reply:
Oh, my fault, sure!

Reply:
Should it be still string array, then? Maybe string list instead? 🤔

Reply:
Probably `string array` but why not both? 😄

Reply:
Since everywhere list is favoured (except special case of `[<ParamArray>]` of course) in your library, I did it using list. But can be change in a minute if you want.